settings/table-string.php

```diff
@@ -196,14 +196,16 @@ protected function get_default_primary_column_name() {
 	 *
 	 * @since 2.6
 	 *
-	 * @param PLL_MO[] $mos An array of PLL_MO objects.
-	 * @param string   $s   Searched string.
+	 * @param PLL_Language[] $languages An array of language objects.
+	 * @param string         $s         Searched string.
 	 * @return string[] Found strings.
 	 */
-	protected function search_in_translations( $mos, $s ) {
+	protected function search_in_translations( $languages, $s ) {
```

Comment:
I missed that the array of `PLL_MO` was used here. I thought that this array was something stupid that I made but finally it was not. It's ok as is now thanks to the cache introduced previously.

Reply:
Having the new cache in mind, I though it was worth the change because it simplified the code in `prepare_items()` as well IMHO.